Refine ECE SUSE host configuration page (#6598)
elastic/docs-content: deploy-manage/deploy/cloud-enterprise/configure-host-ubuntu.md

# Configure an Ubuntu host [ece-configure-hosts-ubuntu]

> Use the steps on this page to prepare an Ubuntu server for {{ece}} (ECE): install Docker from the upstream Docker `apt` repository, set up an XFS volume for cluster data, tune kernel and systemd parameters for production workloads, and pin the Docker version so unattended upgrades don't break ECE.

Reviewer: ```suggestion
Use the steps on this page to prepare an Ubuntu server for {{ece}} (ECE): install and configure Docker, set up an XFS volume for ECE data, tune kernel and systemd parameters for production workloads, and pin the Docker version to avoid unexpected upgrades.
```

Author: 👍 fixed in 2550b484af6f48f957b5c221450cee63812f5cc2

      $(lsb_release -cs) stable" | sudo tee /etc/apt/sources.list.d/docker.list > /dev/null
    ```

4. Install the correct version of the `docker-ce` package. The following command is an example of installing Docker {{ece-docker-version}}. To install a different Docker version, replace {{ece-docker-version}} with your preferred version from the [Support matrix](https://www.elastic.co/support/matrix#elastic-cloud-enterprise).

    ```sh subs=true

>     sudo apt update && sudo apt install -y docker-ce=5:{{ece-docker-version}}.* docker-ce-cli=5:{{ece-docker-version}}.* containerd.io

Reviewer 2: @eedugon ~will this `{{ece-docker-version}}` be accidentally replaced by someone if they think it should be an environment variable in docs? If so, I would suggest we use a different way to express this, such as `<ece-docker-version>` or whatever that won't cause doc team's confusion. WDYT?~

Scratch that, it's probably good to use variable to make things easier to maintain, anyways.

Reviewer: Yes, when possible we prefer to use variables, and they will be rendered in the docs not as variables, but with the actual values.

On SuSE 15 SP4 doc we can't, except if we create a new variable for SuSE. In the near future, if we align and start supporting a similar docker version across ALL OSs then we would use a single variable in the 3 docs.
